packages/fuels-core/src/types/wrappers/transaction.rs

```diff
@@ -170,8 +173,14 @@ pub trait GasValidation {
     fn validate_gas(&self, min_gas_price: u64, gas_used: u64) -> Result<()>;
 }
 
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
+pub trait SignableTransaction {
```

Comment:
Do we have some plan on how this is to be used? Don't see a usage in our code currently. Otherwise I'd just make it part of the Transaction trait.

Reply:
The idea was not to make the whole `Transaction` trait depend on `async_trait` because of one method.